lib/oneview-sdk/resource/api200/volume_template.rb

```diff
@@ -38,14 +39,14 @@ def initialize(client, params = {}, api_ver = nil)
       # @raise [StandardError] if the resource creation fails
       # @return [Resource] self
       def create(header = DEFAULT_REQUEST_HEADER)
-        super(header.merge('Accept-Language' => 'en_US'))
+        super(header)
       end
 
       # Deletes the volume template from OneView
       # @param [Hash] header The header options for the request (key-value pairs)
       # @return [TrueClass] if the volume template was deleted successfully
       def delete(header = DEFAULT_REQUEST_HEADER)
-        super(header.merge('Accept-Language' => 'en_US'))
+        super(header)
```

Comment:
If you are following my suggestion I guess you would still need to merge the `header` with the `DEFAULT_REQUEST_HEADER`, otherwise default values may be overwritten.
```Ruby
def create(header = {})
  super(header.merge(DEFAULT_REQUEST_HEADER))
```